src/codegen/object.rs

```diff
@@ -47,6 +47,22 @@ pub fn generate(
             )?;
         }
 
+        if !analysis.builder_properties.is_empty() {
+            // generate builder method that returns the corresponding builder
+            let builder_name = format!("{}Builder", analysis.name);
+            writeln!(
+                w,
+                "
+            // rustdoc-stripper-ignore-next
+            // Create an instance of [`{builder_name}`].
```

Comment:
the comment here could be a bit better, what do you think @Hofer-Julian ?

Reply:
I like this one
https://docs.rs/http/0.2.1/http/request/struct.Request.html#method.builder

Maybe we can formulate it the same way?

Reply:
Much better than what I wrote :-)